### PR TITLE
{bio}[GCC/10.2.0] taxator-tk v1.3.3, SeqAn v2.4.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a8553902d65268e91109aed47eef1c45 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5d9114aa01be0906900ad7b760ac2c1a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/ecc889e195febbd1ee00e90b5cb23022 for a full test report.

File: easybuild/easyconfigs/t/taxator-tk/taxator-tk-1.3.3-gompi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'taxator-tk'
+version = '1.3.3'
+
+homepage = 'https://github.com/fungs/taxator-tk'
+description = "A set of programs for the taxonomic analysis of nucleotide sequence data"
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
```
Comment (branfosj) [inline easybuild/easyconfigs/t/taxator-tk/taxator-tk-1.3.3-gompi-2020b.eb:9]: It does not look like taxator-tk uses MPI, so we can move it to GCC / 10.2.0.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_COUNT=16

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/bd5163490f775a2aeae3a75d6024cacd for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**